<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>851</width>
      <height>491</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
background-color: rgb(16, 16, 16);
background-color: rgb(6, 6, 6);
	background-color: rgb(0, 76, 227);
border-radius:20px;
}</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>10</y>
       <width>851</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame{
background-color: rgb(16, 16, 16);
background-color: rgb(6, 6, 6);
	background-color: rgb(0, 76, 227);
border-radius:20px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="min">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#min{
	background-color: rgb(173, 173, 173);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:6px;

color: white;


}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cerrar">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>0</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#cerrar{
background-color: rgb(255, 0, 0);
border-radius:6px;

font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 0, 125);
color:white;

}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Lucida Console&quot;;
color:white;
font: 90 15pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>HASH </string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 125);
border-radius:20px;</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Lucida Console&quot;;
color:white;
font: 90 15pt &quot;Rockwell Extra Bold&quot;;</string>
       </property>
       <property name="text">
        <string>TABLE</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>240</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/tablaHash/proximo (1).png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(223, 223, 223);
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>          </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Lucida Console&quot;;

color: rgb(0, 0, 115);
;
font: 70 15pt &quot;Rockwell Extra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>TAMAÃ‘O DE LA TABLA</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(223, 223, 223);
border-radius:6px;
color: white</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Lucida Console&quot;;

color: rgb(0, 0, 115);
font: 70 15pt &quot;Rockwell Extra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>TIPO DE HASHING</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>309</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(223, 223, 223);
border-radius:6px;
color: white</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Lucida Console&quot;;

color: rgb(0, 0, 115);
;
font: 70 15pt &quot;Rockwell Extra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>TIPO DE COLISIONES</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../multimedia/iconos.qrc"/>
 </resources>
 <connections/>
</ui>
